lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
     <div class="page-content">
       <nav class="page-content-nav">
         <a href="" class="nav-item is-active">My Teams</a>
-        <a href="" class="nav-item">Browse All Teams</a>
+        <a href="" class="nav-item">Browse</a>
       </nav>
 
       <div class="page-content-main">
-        <p>Blah blah blah. Go Team!</p>
+
+        <div class="search-tools">
+          <span class="search-count">Viewing 12 of 24 topics</span>
+          <span class="search-sort">sorted by
+          <select name="search-sort">
```

Comment:
Since this page is just a non-interactive mockup, I have a note for eng to make this a form - or talk to @cptvitamin if they want to do something non-form like.